diff --git a/node_modules/rollup-plugin-svelte-svg/index.js b/node_modules/rollup-plugin-svelte-svg/index.js
index 22de0a7..c0d26a1 100644
--- a/node_modules/rollup-plugin-svelte-svg/index.js
+++ b/node_modules/rollup-plugin-svelte-svg/index.js
@@ -35,7 +35,7 @@ exports.svelteSVG = function svelteSVG(options = {}) {
 		load(id) {
 			if (!id.endsWith(".svg.svelte")) return null;
 
-			return readFile(id.slice(0, -".svelte".length), "utf-8")
+			return readFile(id.slice(1, -".svelte".length), "utf-8")
 				.then(file => (svgo ? optimise(file, svgo).data : file))
 				.then(file => file.replace(svgheader, "").trim())
 				.then(addProps);
